src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/Modal/feedback-affected-students-modal.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="modal-header">
+    <h4 class="modal-title" [jhiTranslate]="TRANSLATION_BASE + '.header'"></h4>
+    <button type="button" class="btn-close" aria-label="Close" (click)="activeModal.dismiss()"></button>
+</div>
+<div class="modal-body">
+    <table class="table table-striped mb-3">
+        <colgroup>
+            <col class="col" />
+            <col class="col" />
+            <col class="col" />
+        </colgroup>
```

Comment:
Could we also achieve this with `col-n` instead of the `colgroup`, like in the `feedback-analysis.component`? :)